manifests/client.pp

```diff
@@ -264,9 +302,10 @@
     notify      => Exec["generate ${name}.ovpn in ${server}"],
   }
 
+  $name_escaped = regsubst(regsubst($name, '\.', '\\.', 'G'), '@', '\\@', 'G')
```

Comment:
Do I understand this line correctly as that the following replacements will be done?

* `.` => `\.`
* `@` => `\@`